tests/unit/customizations/cloudformation/test_artifact_exporter.py

```diff
@@ -222,11 +227,10 @@ def _helper_verify_export_resources(
     assert result == expected_result
 
 
-class TestArtifactExporter(BaseYAMLTest):
```

Comment:
I'm not sure if we should be switching the base class here as the current base class helps avoid mutating global state.